react/components/FilterBar/FilterTag.js

```diff
@@ -155,6 +155,38 @@ class FilterTag extends PureComponent {
       ? false
       : options[subject].verbs.length === 1
 
+    const compatibleOptions = {}
+    Object.keys(options).forEach(opt => {
```

Comment:
`Object.keys` do not grant the order. Could that be an issue here?

Reply:
no problem